source/documentation/obligationsandreturns.html.md

```diff
@@ -305,52 +350,52 @@ Software must send the calculationId that matches the calculation the customer i
 
 ## View previously submitted updates
 
-A customer may want to retrieve previously submitted data, for example before making a change the customer may want to request the last update provided before sending in any changes.  If the customer has recently started using your software, you may need to retrieve previous data.
+A customer may want to retrieve previously submitted data, for example before making a change the customer may want to request the last update provided before sending in any changes. If the customer has recently started using your software, you may need to retrieve previous data.
 
 ### Periodic updates
 
-Software can use the [self employment or property periodic updates](https://developer.service.hmrc.gov.uk/api-documentation/docs/api/service/self-assessment-api/2.0#selfemployment-business) endpoint to retrieve the list of updates made for that income source, or to find one or more period IDs.
-The period ID is then used with the ‘get a self employment/property (FHL or Non FHL) periodic update’ endpoint to retrieve data for that update.
+Software can use the list all self employment or property update periods endpoints to retrieve the list of updates made for that income source, or to find one or more period IDs. The period ID is then used with the ‘get a selfemployment/property (FHL or Non FHL) periodic update’ endpoint to retrieve data for that update.
```

Comment:
Space/hyphen in "selfemployment"?

Reply:
minor typos like this are expected at this stage